deploy/13-verification.ts

```diff
@@ -0,0 +1,34 @@
+// @ts-nocheck
+import  "@nomiclabs/hardhat-ethers";
+import hre from 'hardhat';
+import fs from 'fs';
+import path from 'path';
+
+async function main(){
+    // define the network that you want to verify the contracts
+    const network = "fuji"
```

Comment:
Can we set const network = hre.network.name; instead of const network = "fuji" ?

Reply:
done